persimmon/persimmon/settings.py

```diff
@@ -12,32 +12,36 @@
 
 from pathlib import Path
 
+import os
+
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = ')dimn^b^qcbpf8stkoinv#5b$&_@@y4!+2v_zys5pt&8ky1ade'
-
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
 ALLOWED_HOSTS = ['testserver', '127.0.0.1']
 
+os.environ['DJANGO_SETTINGS_MODULE'] = "persimmon.settings"
```

Comment:
You should remove this line and the os import; it's bad practice - the environment is supposed to be configurable

Reply:
Done.